CONTRIBUTING.md

```diff
@@ -325,7 +325,7 @@ Note that for VSCode, to enable "live" linting of TypeScript (and other) file ty
   "eslint.autoFixOnSave": true,
 ```
 
-It is **not** recommended to use `prettier` plugin on Kibana project. Because settings are in `eslintrc.js` file and it is applied to too many files that shouldn't be prettier-ized. 
+:warning: It is **not** recommended to use the [`Prettier` extension/IDE plugin](https://prettier.io/) while maintaining the Kibana project. Formatting and styling roles are set in the multiple `.eslintrc.js` files across the project and some of them use the [NPM version of Prettier](https://www.npmjs.com/package/prettier). Using the IDE extension might cause conflicts, applying the formatting to too many files that shouldn't be prettier-ized.
```

Comment:
it almost looks like this `:warning:` prefix is some kind of markdown thing, but AFAIK it isn't.  Would probably be better to change this to `**Warning**:` instead.

Reply:
Regarding your comment about the `:warning:`. My intention was to bring the attention by using this emoji ⚠️ . Do you think we should use another one or `**Warning**` instead? Here's a very complete list with all the possibilities: https://gist.github.com/rxaviers/7360908

Reply:
Ah, I missed that the first time - swore I looked at the rendered output, but see it now, thx!